src/sentry/api/serializers/models/group.py

```diff
@@ -817,10 +820,10 @@ def get_attrs(self, item_list, user):
             stats = partial_get_stats()
             if self.has_dynamic_issue_counts:
                 filtered_stats = (
-                    partial_get_stats(conditions=self.conditions) if self.conditions else stats
+                    partial_get_stats(conditions=self.conditions) if self.conditions else None
                 )
             for item in item_list:
-                if self.has_dynamic_issue_counts:
+                if self.has_dynamic_issue_counts and self.conditions:
```

Comment:
Nit: Could use `filtered_stats is not None` since that's what's more directly used in this statement.